homeassistant/components/twitch/__init__.py

```diff
@@ -46,10 +47,11 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     client.auto_refresh_auth = False
     await client.set_user_authentication(access_token, scope=OAUTH_SCOPES)
 
-    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = {
-        CLIENT: client,
-        SESSION: session,
-    }
+    coordinator = TwitchCoordinator(hass, client, session)
+
+    await coordinator.async_config_entry_first_refresh()
+
+    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = coordinator
```

Comment:
Would it not make sense to just use `entry.runtime_data` as well, when you have to touch the lines anyway?

Reply:
Wanted to avoid making this PR bigger than needed, so I did not do that right now

Reply:
Yeah, okay, I just thought because in regards to line count it wouldn't make a difference :smile: But fair